apps/api/src/database/seed/data-test.js

```diff
@@ -479,6 +483,47 @@ export async function seedTestData(databaseConnector) {
 		});
 	}
 
+	const linkedAppealGroups = [
+		[appeals[0].id, appeals[1].id, appeals[2].id],
```

Comment:
We could test for those array indexes but not a big deal as we're aware of the size of the test data is the seed...